Octokit.Reactive/IObservableGitHubClient.cs

```diff
@@ -18,6 +18,8 @@ public interface IObservableGitHubClient : IApiInfoProvider
 
         IObservableAuthorizationsClient Authorization { get; }
         IObservableActivitiesClient Activity { get; }
+        IObservableApplicationClient Application { get; }
+        IObservableInstallationsClient Installations { get; }
```

Comment:
We use singular for client property names, so can this be `Installation` ?

Reply:
But I see there is `AhtorizationsClient` and `ActivitiesClient`, in fact only `ApplicationClient` is in singular form...

Reply:
Oh, sorry, I see now what you meant. Will fix.